### PR TITLE
Add troubleshoot sections in "guides"

### PR DESCRIPTION
**Pull request by bhawnasehgal**
Hey @lorforlinux 

I have added 3 different sections for troubleshooting:

1. Tips for better signal acquisition
2. Fixing hardware setup
3. Resolving software issues

Please review, and merge.

### CODE REVIEW

File: guides/troubleshoot/resolve-software-issues/index.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Resolving software issues
+#############################
+
+:bdg-info:`Arduino IDE`
+
+- **Better signal visualization:** Many people find it difficult to visualize the biopotential signals in the serial plotter even if everything is correct because the serial plotter of the latest version (v2.0) of Arduino IDE plots very few data points per frame and this is why you may find it moving very fast. To resolve this issue you can either:
+  
+  - Download the Arduino IDE version 1.8.X (Legacy IDE) in which the serial plotter plots more data points per frame so it moves comparatively slow as compared to the latest version.
+  - Use `Chords application <chords.upsidedownlabs.tech>`_ to visualise the data.
```
Comment (lorforlinux) [inline guides/troubleshoot/resolve-software-issues/index.rst:11]: I think we should only recommend Chords application.

File: guides/usage-guides/using-gel-electrodes/index.rst [hunk 2]
```diff
@@ -100,6 +100,30 @@ Connect the BioAmp Snap/Alligator Cable on the gel electrodes.
 
 Peel of the plastic backing and place the electrodes on your targeted skin surface as per the connections directed with every BioAmp hardware.
 
+.. figure:: media/electrode-placement-eog.*
+    :align: center
+    :alt: Electrode placement for EOG
+
+    *Gel electrodes placement for EOG signal recording*
+
```
Comment (lorforlinux) [inline guides/usage-guides/using-gel-electrodes/index.rst:108]: Why are we adding `*` with this text?

Comment (bhawnasehgal) [reply guides/usage-guides/using-gel-electrodes/index.rst]: To make the text below image in italics. But reverting it back to normal formatting.

**Comment by lorforlinux** [comment]
@bhawnasehgal thank you for your contributions, I am merging this now.

**Pull request merged**